source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProLinesViewModel.cs

```diff
@@ -58,11 +58,36 @@ public LineFromTypes LineFromType
             }
         }
 
+        LineTypes lineType = LineTypes.Geodesic;
+        public LineTypes LineType
+        {
+            get { return lineType;  }
+            set
+            {
+                lineType = value;
```

Comment:
This probably isn't called very often (but I'm not sure how the view even hooks up to this) so probably doesn't need a guard:
if (value == lineType) return